### ISSUE TITLE
ParentalKey with unique set to True should only allow adding of one inline in the admin

### ISSUE THREAD

**Issue by jmiguelv** [torchbox/wagtail#198]
When creating a model with ParentalKey, with unique=True, the admin should only allow adding one inline of that model, but currently it allows adding multiple.

Example model:
```python
class FundsRaised(TimeStampedModel):
    pageant = ParentalKey('Pageant', related_name='funds_raised', unique=True)
    object_of_funds_raised = models.CharField(max_length=128,
                                              verbose_name='Object')
    notes = RichTextField(blank=True)
```

Admin screen:
![Admin screenshot](https://cloud.githubusercontent.com/assets/1233186/2710198/5eba1406-c4be-11e3-981f-f1d047eac3e5.png)

**Comment by kaedroho** [comment]
Validation added in #1529. Would need some FE work to prevent too many panels being created

**Comment by gasman** [comment]
Fully implemented (including front-end work) in #1529.
